app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java

```diff
@@ -28,6 +28,7 @@
 import android.text.InputType;
 import android.text.TextUtils;
 import android.text.TextWatcher;
+import android.util.AttributeSet;
```

Comment:
@rajdeep1008 It is not an important issue as such, but i pointed this out for you to be extra careful from now onwards.